filebeat/module/haproxy/log/_meta/fields.yml

```diff
@@ -32,10 +33,12 @@
     - name: captured_headers
       description: >
         List of headers captured in the request due to the presence of the "capture request header" statement in the frontend.
+      # Do we need text? does this even have be be indexed?
       type: text
 
     - name: raw_request_line
       description: Complete HTTP request line, including the method, request and HTTP version string.
+      # keyword? sounds like a .original field but none is matching
```

Comment:
Not quite. This module doesn't parse the HTTP request. This line contains a full HTTP request: `GET /foo HTTP/1.1`. So would leave here for now. But agree to move to keyword.

Reply:
https://github.com/elastic/beats/pull/10397